db/data/20221011111730_fix_incorrect_draft_findable_courses.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class FixIncorrectDraftFindableCourses < ActiveRecord::Migration[7.0]
+  def up
+    Course.joins(:site_statuses, :enrichments, provider: :recruitment_cycle).where(provider: { recruitment_cycle: { year: "2023" } }, enrichments: { status: %w[draft rolled_over] }, site_statuses: { status: "running" }).find_each do |course|
+      next if course.is_published?
+
+      course.site_statuses.update_all(publish: :unpublished, status: :new_status)
```

Comment:
```suggestion
      vac_status = determine_the_right_thing

      course.site_statuses.update_all(publish: :unpublished, status: :new_status, vac_status: vac_status)
```

https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/models/site_status.rb#L72-L83